js/Common.js

```diff
@@ -26,6 +26,19 @@ function SetupArrayRandomFunction(){
 			}
 }
 
+// Return a random element of the array if it's not equal to previuousValue
+function SetupArrayNonRepeatingRandomFunction(){
+	Array.prototype.notRepeatingRandom = function (previousValue) {
```

Comment:
If an array is empty/null/undefined - return previous value, and write error to console.log().